app/src/main/res/layout/activity_gym_preset.xml

```diff
@@ -1,50 +1,44 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".GymActivity">
+    android:orientation="vertical"
+    android:gravity="center"
+    tools:context=".GymPresetActivity">
 
-    <Button
-        android:id="@+id/gym_confirm_button"
-        android:layout_width="wrap_content"
+    <TextView
+        android:id="@+id/gymPresetTextView"
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_marginStart="160dp"
-        android:layout_marginLeft="160dp"
-        android:layout_marginBottom="48dp"
-        android:onClick="onGymConfirmClick"
-        android:text="Confirm"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintStart_toStartOf="parent" />
+        android:layout_marginTop="30dp"
+        android:textSize="20sp"
+        android:textStyle="bold"
+        android:gravity="center_horizontal"
+        android:text="TextView" />
 
     <TextView
-        android:id="@+id/gymPresetTextView"
-        android:layout_width="411dp"
+        android:id="@+id/availableEquip"
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_marginTop="88dp"
+        android:layout_marginTop="41dp"
+        android:textSize="16sp"
         android:gravity="center_horizontal"
-        android:text="TextView"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+        android:text="Available Equipment" />
 
     <ListView
         android:id="@+id/gymPresetListView"
         android:layout_width="395dp"
         android:layout_height="460dp"
-        android:layout_marginStart="8dp"
-        android:layout_marginLeft="8dp"
-        android:layout_marginTop="8dp"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/availableEquip" />
+        android:gravity="center_horizontal"
+        android:layout_marginTop="8dp" />
 
-    <TextView
-        android:id="@+id/availableEquip"
-        android:layout_width="412dp"
+    <Button
+        android:id="@+id/gym_confirm_button"
```

Comment:
Might want to add this back 
`android:layout_marginBottom="48dp"`

Reply:
That was due to absolute positioning margins but I replaced them with relative positioning. Hopefully it works on all devices now.